app/assets/stylesheets/stories.scss

```diff
@@ -81,6 +101,25 @@
         word-break: break-all;
       }
     }
+
+    em {
```

Comment:
I'd say to move these styles to reuse the same style set for `.story-description` above, it's the same style, maybe doing `.story-description, .story_preview` in line 13.